common/persistence/src/test/java/app/coronawarn/server/common/persistence/repository/DiagnosisKeyRepositoryTest.java

```diff
@@ -1,13 +1,14 @@
 package app.coronawarn.server.common.persistence.repository;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
The assert statements are coming from junit4, please use the ones that are coming from junit5 (org.junit.jupiter)